### PR TITLE
Milestone: Application Deploy

### PR DESCRIPTION
**Pull request by Whoistolu**
## Application Deploy
In this milestone, we:
- Deployed our application using gtihub pages
- We added the live link into our readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ This milestone consists of two (2) sections:
 
 ## Live Demo
 
-[Live Demo Link](https://cranky-haibt-bf176a.netlify.app/)
+[Live Demo Link](https://stupefied-payne-1fc0b5.netlify.app/)
```
Comment (oxenprogrammer) [inline README.md]: *Changes Requested*
- [ ] Well done deploying a live version of the project to netlify. However, it is required that the project is deployed rather using Github Pages. Let us make these changes and move forward.

deploy

**Review by oxenprogrammer — CHANGES_REQUESTED** [review]
Hi @Whoistolu , 

Good job so far!
I can see you're really putting effort to become a world-class software developer :+1:
There are some issues that you still need to address in order to move forward to the next project.
Trust me you got this! :muscle:


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Consider the optional changes nice to have features and only work on them when you have satisfied the required changes_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: README.md (hunk 1 above)
Comment (Whoistolu) [reply README.md]: Hello @oxenprogrammer, I deployed using github pages initially but the live link generated has refused to load the images I have on my project. Is there a way forward to this?
Kindly take a look at it. https://whoistolu.github.io/personal-portfolio/

Comment (Whoistolu) [reply README.md]: I have fixed it up. It now works fine.

**Pull request merged**